src/wooden-tree/defaultStore.js

```diff
@@ -0,0 +1,52 @@
+import Tree, { ActionTypes } from './';
+
+const actionMapper = {
+  [ActionTypes.EXPANDED]: Tree.nodeExpanded,
+  [ActionTypes.DISABLED]: Tree.nodeDisabled,
+  [ActionTypes.SELECTED]: Tree.nodeSelected,
+  [ActionTypes.CHILD_NODES]: Tree.nodeChildren,
+  [ActionTypes.LOADING]: Tree.nodeLoading,
+};
+
+export default (state = null, action) => {
```

Comment:
I think that initial state should be something else than null. Empty object probably (or some structure that will work).